pow-migration/src/main.rs

```diff
@@ -71,12 +72,11 @@ fn initialize_logging() {
 
 #[tokio::main]
 async fn main() {
-    //    1 - Use the monitor library to send ready txn and determine if enough validators are ready
-    //    2 - Once enough validators are ready we select the next genesis candidate and wait until that block is mined
-    //    3 - When the genesis candidate is mined we start the genesis generation process
-    //    4 - Monitor the PoW chain to detect if the genesis candidate is forked
-    //    5 - After X confirmations, start the 2.0 client with the generated genesis block
-    //    6 - If a fork is detected, go back to step 3 and repeat
+    //    1 - Wait for the pre-stake window to close (by waiting the confirmations).
```

Comment:
Shouldn't we allow to pro-actively migrate history, even if the pre-stake window is not closed ?

Reply:
Yes, code does it. Fixed comments